src/request.ts

```diff
@@ -53,8 +52,11 @@ export const fetchJSON = (
 };
 
 const hasArgumentsError = (responseData: JSONResponseData): boolean =>
-    isPlainObject(responseData['error_detail']) &&
-    isPlainObject(responseData['error_detail']['ARGUMENTS_ERROR']);
+    !!(
+        responseData &&
```

Comment:
you don't need to check this with typescript right? i assume you'd get a warning otherwise since responseData is required

Reply:
So I don't need to check the existence of `responseData` because of TypeScript, but I do need to check the existence of of the sub properties since they are the response from the server. Will fix!